.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: java
 script:
-- ./gradlew check
+  - ./gradlew check --info
```

Comment:
Nice, thanks @alkemist! I'll include it in this PR then
`started` makes more sense than `passed` for us I guess, since we care about the skipped ones being run as well.